mu-plugins/blocks/global-header-footer/postcss/header/menu.pcss

```diff
@@ -308,7 +308,7 @@
 	& .wp-block-navigation__container {
 
 		@media (--tablet) {
-			background: var(--wp-global-header--background-color);
+			background: inherit;
```

Comment:
Rather than changing this, you can add another "case" to `_common.pcss` to override `--wp-global-header--background-color`. See how it's done for the other styles here: https://github.com/WordPress/wporg-mu-plugins/blob/2ca5e5ba0046f5fa384a504ffe07df0b2b29a6c4/mu-plugins/blocks/global-header-footer/postcss/_common.pcss#L67-L90